source/tests/unit/ni_fake_service_tests.cpp

```diff
@@ -161,7 +161,7 @@ TEST(NiFakeServiceTests, NiFakeService_InitWithOptionsAndResetServer_SessionIsCl
   const char* session_name = "sessionName";
   EXPECT_CALL(library, InitWithOptions)
       .WillOnce(DoAll(SetArgPointee<4>(kTestViSession), Return(kDriverSuccess)));
-  EXPECT_CALL(library, close(kTestViSession))
```

Comment:
Note: All the tests fixes were because the names changed in the metadata when updated from hapigen. Some were our doing intentional (name overrides for enums/attributes) but some were just the nature of updating the version of the ni-fake hapigen metadata we're based on (new fields, casing changes, etc.)